data/scripts/dependency-resolver.py

```diff
@@ -59,6 +59,7 @@ def handle_pkgconfig_check(args, conf):
     max_ver = conf.get("max-version")
     exact_ver = conf.get("exact-version")
     ver_match = True
+    required = True if conf.get("required") else False
```

Comment:
ugly as Perl.

Python version is: required = conf.get("required", False)

Also, make sure you're using the JSON boolean (true, false) not a string!